presto-client/pom.xml

```diff
@@ -75,10 +75,19 @@
             <scope>test</scope>
         </dependency>
 
+        <!--  TODO: Kindly suggest if another library is preferred. -->
+        <dependency>
```

Comment:
Can we use TestNG ?

Reply:
Hi Praveen,
I am using TestNG along with Mockito for mocking the dependencies. I did find a reference to mockito library in presto-bigquery as well so thought of adding it though if another library is preferred for mocking then I would be happy to replace it.  Let me know if you are good with this. Thank you!